### PR TITLE
Remove sentry and related code

### PR DESCRIPTION
**Pull request by willdurand**
Fixes https://github.com/mozilla/addons-frontend/issues/10028

---

I removed a component that was "mainly used to test the Sentry integration" in addition to the configs and related code.

### CODE REVIEW

**Comment by eviljeff** [comment]
Did we actually come to a final decision on removing sentry, and what would be a replacement for tracing errors?

**Comment by willdurand** [comment]
> Did we actually come to a final decision on removing sentry

I thought we did, yeah.

> and what would be a replacement for tracing errors?

- (server) logs
- as for other errors, it's usually easy because either the app works or is terribly broken

**Comment by eviljeff** [comment]
> > Did we actually come to a final decision on removing sentry
> 
> I thought we did, yeah.

I remember a few different conversations but I may have missed some threads.  @diox to weigh in.

> > and what would be a replacement for tracing errors?
> 
>     * (server) logs
>
>     * as for other errors, it's usually easy because either the app works or is terribly broken

I'm not convinced server logs are an improvement or it's easy, but 🤷‍♂️

**Comment by diox** [comment]
:+1: on removing sentry from addons-frontend, there is too much noise and we don't actively use it.

File: webpack-common.js [hunk 1]
```diff
@@ -135,7 +132,6 @@ export function getPlugins({ excludeOtherAppLocales = true } = {}) {
     new webpack.DefinePlugin({
       CLIENT_CONFIG: JSON.stringify(clientConfig),
       'process.env.NODE_ENV': JSON.stringify('production'),
-      DEPLOYMENT_VERSION: JSON.stringify(getDeploymentVersion({ versionJson })),
```
Comment (eviljeff) [inline webpack-common.js:138]: this was only used for senty too?

**Review by eviljeff — APPROVED** [review]

File: webpack-common.js (hunk 1 above)
Comment (willdurand) [reply webpack-common.js:138]: apparently yes

**Pull request merged**